xtask/src/docgen.rs

```diff
@@ -48,6 +54,67 @@ pub fn typable_commands() -> Result<String, DynError> {
     Ok(md)
 }
 
+pub fn static_commands() -> Result<String, DynError> {
+    let mut md = String::new();
+    let keymaps = [
+        EditorView::default().keymaps.map()[&Mode::Normal].reverse_map(),
+        EditorView::default().keymaps.map()[&Mode::Select].reverse_map(),
+        EditorView::default().keymaps.map()[&Mode::Insert].reverse_map(),
+    ];
+
+    md.push_str(&md_table_heading(&[
+        "Name".to_owned(),
+        "Description".to_owned(),
+        "Default keymaps".to_owned(),
+    ]));
+
+    for cmd in MappableCommand::STATIC_COMMAND_LIST {
+        let keymap_strings: Vec<String> = keymaps
+            .iter()
+            .enumerate()
+            .map(|(_, keymap)| {
+                keymap
+                    .get(cmd.name())
+                    .map(|bindings| {
+                        bindings.iter().fold(String::new(), |mut acc, bind| {
+                            if !acc.is_empty() {
+                                acc.push_str(", ");
+                            }
+                            for key in bind {
+                                acc.push_str(&key.key_sequence_format());
+                            }
```

Comment:
```suggestion
                            acc.push_str("`` ");
                            for key in bind {
                                acc.push_str(&key.key_sequence_format());
                            }
                            acc.push_str(" ``");
                            acc

```

To use inline code styling for the keybindings we can use a markdown trick. You can use double backticks to allow single backticks in a code snippet.

```markdown
switch_to_lowercase: `` ` ``
```

renders as

> switch_to_lowercase: `` ` ``

for example. That will also take care of things that look like HTML tags like `<left>`/`<right>`